File: redsheet-portal/src/app/database-models/v-user-lookup.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vUserLookup {
    constructor(vUserLookup?: vUserLookup )  {
        let properties = [ "companyId", "email", "requestorId", "userId", "userName",];
        for (let property of properties)
            if (vUserLookup && vUserLookup[property])
                this[property] = vUserLookup[property];
    }
 
    @required()
	companyId : string =   undefined;
 
    @required()
    @maxLength(150)
	email : string =   undefined;
 
    @required()
	requestorId : string =   undefined;
 
	userId : number =   0 ;
 
    @required()
    @maxLength(201)
	userName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vUserLookup.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vUserLookups", Schema = "dbo")]
	[Serializable]
	public partial class vUserLookup  
	{
 
	[Required]
	public Guid CompanyId { get; set; } 
 
	[Required] 
	[MaxLength(150)]
	public string Email { get; set; } 
 
	[Required]
	public Guid RequestorId { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int UserId { get; set; } 
 
	[Required] 
	[MaxLength(201)]
	public string UserName { get; set; } 

	public vUserLookup()
	{
    }

    }
}

